xml/ReportingOrganisation.xml

```diff
@@ -2563,6 +2818,7 @@
             <code>US-EIN-522043740</code>
             <name>
                 <narrative>Discovery Learning Alliance, Inc.</narrative>
+                <narrative xml:lang="fr">Impact(Ed) International</narrative>
```

Comment:
This one’s a bit surprising… I suspect Discovery Learning Alliance is the old name, and the French translation here is actually the new name.